### DISCUSSION TITLE
Qfield asks for a Main Password after logging in

### DISCUSSION THREAD

**Discussion by yban06** [opengisch/QField#2546] [Q&A]
Hi,

Friend of mine tries to log in qfield and gets after logging in a new message asking for a "PRESENT" Main password. See image below. Does that has something to do with Qfield or does this refer to a main password manager on his pc? 

Regards
Yves
![2022-03-01 15_58_45-Start](https://user-images.githubusercontent.com/100617686/156192926-22b8d3d4-fed9-48c2-8a87-e2774397968f.jpg)

**Comment by juwizeye** [comment]
they are asking for Qfieldcloud  password .

**Comment by suricactus** [comment]
@yban06 Yes, QFieldSync is asking access to the password manager so it can store your password securely on your PC. 

The password is store as securely as your passwords with PostGIS/WFS/WMS etc password protected services.

This master password is configured only once per QGIS profile and after that it logins automatically.

**Reply by yban06** [reply]
Hi Suricactus and Juwizeye, 

Thanks for your replies and help. We'll try this, 
Regards,
Yves
